politico/api/v1/user/routes.py

```diff
@@ -92,37 +92,45 @@ def update(id):
     print(user)
     if not user:
         return make_response(jsonify({
-            'status': 406, 
+            'status': 404, 
             'error': 'No user with id:{} was found'.format(id)
-        }), 406)
+        }), 404)
     else:
         updated_user = user_table.update_user(id, user_data)
-        print('updated****')
-        return make_response(jsonify({
-            'status': 201,
-            'data': [updated_user]
-        }), 201)
+        if not updated_user:
+            return make_response(jsonify({
```

Comment:
don't return 500 errors, catch the specific error and return appropriate message